poster/poster.tex

```diff
@@ -77,9 +77,28 @@
 
 \begin{block}{Reverse your program}
 	\begin{itemize}
-		\item Division example
-		\item Show inverse side-by-side
+
+		\item The inverse of a program can be computed automatically.
+		\center
+		\textsc{\small Division}\\ \vspace{1cm}
+		\begin{minipage}{.8\textwidth}
+		\code{divide}
+		\end{minipage}
+		\vspace{1cm}
+		\item \textsc{\small Reverse of division}\\ 			\textsc{\tiny{automatically computed}} \\ \vspace{1cm}
```

Comment:
Better to not repeat that its automatically computed (you mention it in the text above. If you'd like, put `(for free!)` next to `\small Reverse of division`